module/variable.go

```diff
@@ -0,0 +1,18 @@
+package module
+
+import (
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/hcl/v2"
+	"github.com/zclconf/go-cty/cty"
+)
+
+type Variable struct {
+	Description     string
+	Type            cty.Type
+	TypeConstraints schema.ExprConstraints
+	Default         hcl.Expression
```

Comment:
I reckon we'll need this eventually for https://github.com/hashicorp/terraform-ls/issues/495 - good call adding it, but I think we could already store it as `cty.Value` and call the field `DefaultValue`?

Reply:
Didn't know cty.Value did exist 😁changing it to cty.Value